emcc

```diff
@@ -708,7 +708,7 @@ try:
   if js_opts is None: js_opts = opt_level >= 2
   if llvm_opts is None: llvm_opts = LLVM_OPT_LEVEL[opt_level]
   if opt_level == 0: debug_level = max(3, debug_level)
-  if memory_init_file is None: memory_init_file = opt_level >= 2
+  if memory_init_file is None: memory_init_file = 2 if opt_level >= 2 else 0
```

Comment:
Yes, it would make this the default. I would have assumed that for most applications, the overhead for a second request would be bigger than the performance cost of parsing that initializer. But I guess you know your customers better than I do, so if you say that for most their setups mem files are preferable, then I won't argue with that. I know that I personally will try to use the string initializer simply because I prefer to have a single artifact containing the whole program, but that's just a matter of taste to me.